samza-test/src/test/scala/org/apache/samza/test/harness/AbstractIntegrationTestHarness.scala

```diff
@@ -17,10 +17,13 @@
  * under the License.
  */
 package org.apache.samza.test.harness
-import java.util.Properties
+import java.util.{Properties, UUID}
```

Comment:
nit: why adding UUID here? I don't see the usage of UUID in this file.

Reply:
Thanks for the detailed review. I have updated the patch to remove the UUID.